### PR TITLE
Added program area 2384

### LINKED ISSUE #2384: Add Program Area to Jobs for Hope Project Details
### Overview
We need to add program area information to the Jobs for Hope project so that it will show up when we add the program areas filter.

### Action Items
- [ ] Review the project.md file model in the Resources section
- [x] Add the following text to _projects/jobs-for-hope.md
```
program-area:  
    - Social Safety Net
```

### Resources/Instructions
File location to make the change: _projects/jobs-for-hope.md

URL for UAT to check the page has been changed: https://github.com/hackforla/website/blob/gh-pages/_projects/jobs-for-hope.md

[Model for project.md files](https://github.com/hackforla/website/wiki/project.md-file-template)

### PR DESCRIPTION
**Pull request by KazushiR**
Fixes #2384

### What changes did you make and why did you make them ?

added the program-area as instructed

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
image

</details>

### CODE REVIEW

**Comment by SAUMILDHANKAR** [comment]
Review ETA: End of day 2/18/22
Availability: Now for an hour

File: _projects/spare.md [hunk 1]
```diff
@@ -9,8 +9,7 @@ alt-hero: ''
 links:
   - name: GitHub
     url: 'https://github.com/hackforla/spare'
-  - name: Site
-    url: 'http://whatcanyouspare.org'
```
Comment (SAUMILDHANKAR) [inline _projects/spare.md]: @KazushiR Changes made to this file should not be part of this PR.

**Review by SAUMILDHANKAR — CHANGES_REQUESTED** [review]
@KazushiR Thanks for working on this issue. Changes made to the spare.md file should not be part of this PR. Following stpes might fix it: update your forked gh-pages and merge your forked gh-pages into your topic branch. Also, request you to add your ETA and availability while picking up a new issue. Thanks.

**Comment by KazushiR** [comment]
> @KazushiR Thanks for working on this issue. Changes made to the spare.md file should not be part of this PR. Following stpes might fix it: update your forked gh-pages and merge your forked gh-pages into your topic branch. Also, request you to add your ETA and availability while picking up a new issue. Thanks.

@SAUMILDHANKAR 
Ah my mistake. I will fix this issue and make another pull request.

**Comment by SAUMILDHANKAR** [comment]
@KazushiR No worries. There is no need to make another pull request. You can make the changes in this branch itself and add another commit.

**Comment by KazushiR** [comment]
> @KazushiR No worries. There is no need to make another pull request. You can make the changes in this branch itself and add another commit.

@SAUMILDHANKAR Hello, I believe I have made the requested changes. Are you able to confirm on your side?

**Comment by SAUMILDHANKAR** [comment]
@KazushiR Sorry if I was not clear about making the changes to the spare.md file. In your last commit, spare.md file seems to be deleted. This should not be the case and the file should be part of you branch. 
We need to resolve the conflicts in your branch. Please go through the following link, [resolving merge conflict](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) for the possible next steps. Thank you.

**Comment by Carlos-A-P** [comment]
Availability: 2hrs
ETA: Wed, 2/23/2022

**Comment by Carlos-A-P** [comment]
@KazushiR While you're resolving the merging conflicts, also make sure to add "No visual changes" under "Screenshots of Proposed Changes Of The Website"

**Comment by KazushiR** [comment]
> @KazushiR Sorry if I was not clear about making the changes to the spare.md file. In your last commit, spare.md file seems to be deleted. This should not be the case and the file should be part of you branch. We need to resolve the conflicts in your branch. Please go through the following link, [resolving merge conflict](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) for the possible next steps. Thank you.

@SAUMILDHANKAR Hello, I followed the steps and it seems as though I'm not authorized to merge a pull request.

**Comment by KazushiR** [comment]
> @KazushiR While you're resolving the merging conflicts, also make sure to add "No visual changes" under "Screenshots of Proposed Changes Of The Website"

**Comment by Carlos-A-P** [comment]
@KazushiR Once you think your PR is ready to be reviewed, request @SAUMILDHANKAR to review it again. You have to be part of the merging team to merge PRs

**Comment by SAUMILDHANKAR** [comment]
Review ETA: End of day 2/24/22
Availability: Thursday 9-10 PM Pacific Time

**Review by SAUMILDHANKAR — CHANGES_REQUESTED** [review]
@KazushiR Thank you for making another commit to fix the issue, however, as you can see in the files changed tab spare.md is showing up as deleted. Please make sure only the changes made to jobs-for-hope.md file are visible in the files changed tab. Thanks.

**Comment by SAUMILDHANKAR** [comment]
@KazushiR Hope you are doing well. Just wanted to share one of the decisions made during our team meeting. We are now requesting all website team members to attend at least one meeting in a week (Tuesday, Thursday or Sunday) as we strive to replicate a professional working environment within our team amongst other things. Therefore, request you to let us know which of the weekly meetings would be suitable for you based on availability. We have decided to make an exception for this requirement only on a case-by-case basis. I will leave a message for you on slack as well. Thank you.

**Comment by SAUMILDHANKAR** [comment]
Progress update: Kazushi will be back next week.

**Comment by KazushiR** [comment]
> Progress update: Kazushi will be back next week.

Hello, I am feeling better and will start working on this issue this weekend. I apologize on the delay.

**Review by KazushiR — COMMENTED** [review]
I believe I have corrected the changes on this PR

**Comment by SAUMILDHANKAR** [comment]
> I believe I have corrected the changes on this PR

@KazushiR I still see spare.md file as deleted in the files changed tab. Did you try adding it back to your branch?

**Comment by KazushiR** [comment]
> > I believe I have corrected the changes on this PR
> 
> 
> 
> @KazushiR I still see spare.md file as deleted in the files changed tab. Did you try adding it back to your branch?

Hello Saumil. I tried to make the changes in my original branch but when I tried to push the changes, it will make a new PR. Did you want me to leave spare.md in the changes?

**Comment by SAUMILDHANKAR** [comment]
@KazushiR We will need to resolve the merge conflicts which are part of this PR (this might be because changes were made to spare.md file twice). Since, you are being prompted to make a new PR while doing a commit, I am not sure if git revert would work either. But we can explore `git reset --hard`. Please let me know if you would like to go over it during meeting time or some other time that works for you. Thank you.

**Comment by KazushiR** [comment]
> @KazushiR We will need to resolve the merge conflicts which are part of this PR (this might be because changes were made to spare.md file twice). Since, you are being prompted to make a new PR while doing a commit, I am not sure if git revert would work either. But we can explore `git reset --hard`. Please let me know if you would like to go over it during meeting time or some other time that works for you. Thank you.

Hello Saumil. I will look into this further and if I cannot get it resolved, I will go over it with you during the meeting.

**Review by SAUMILDHANKAR — CHANGES_REQUESTED** [review]
@KazushiR Thanks a lot for working on this issue and resolving merge conflicts. Thanks @JessicaLucindaCheng for helping us in resolving merge conflicts. I just noticed that there would be visual changes to the website (as program area: social safety net should be visible in jobs for hope project card) and I should have mentioned it yesterday as well (my bad). So, requesting to add the images in your initial comment. Rest all looks good. Please request a re-review once you have done that. Really appreciate your great work. Thank you.

**Comment by KazushiR** [comment]
> @KazushiR Thanks a lot for working on this issue and resolving merge conflicts. Thanks @JessicaLucindaCheng for helping us in resolving merge conflicts. I just noticed that there would be visual changes to the website (as program area: social safety net should be visible in jobs for hope project card) and I should have mentioned it yesterday as well (my bad). So, requesting to add the images in your initial comment. Rest all looks good. Please request a re-review once you have done that. Really appreciate your great work. Thank you.

@SAUMILDHANKAR I will go ahead and get this done to close out this pr. @JessicaLucindaCheng thank you again for the help.

**Review by SAUMILDHANKAR — APPROVED** [review]
@KazushiR Thanks a lot for making the changes and adding the images. I will just remove the line where it says no change to the website. Rest all looks good. Really appreciate all the work that you have done on this issue. Thank you.

**Pull request merged**